common/main/java/com/couchbase/lite/CouchbaseLiteException.java

```diff
@@ -58,11 +59,12 @@ public static CouchbaseLiteException toCouchbaseLiteException(
         int statusCode,
         @Nullable String msg,
         @Nullable Exception e) {
-        // make sure this gets logged
-        if (e != null) { Log.w(LogDomain.DATABASE, "Lite Core exception", e); }
+        // log the LiteCoreException in case the client swallows it.
+        if ((e != null) && (CouchbaseLiteInternal.debugging())) {
```

Comment:
This is starting to look like the developer switch, so other platforms should take notice.  I support it.